### ISSUE TITLE
ModuleNotFoundError: No module named 'slackclient' // when trying to run training

### ISSUE THREAD

**Issue by neuronflow** [MIC-DKFZ/basic_unet_example#7]
```
python3 run_train_pipeline.py
Using torch multi processing
Traceback (most recent call last):
  File "run_train_pipeline.py", line 28, in <module>
    from experiments.UNetExperiment import UNetExperiment
  File "/home/florian/flow/ibunet/experiments/UNetExperiment.py", line 29, in <module>
    from trixi.experiment.pytorchexperiment import PytorchExperiment
  File "/home/florian/.local/lib/python3.6/site-packages/trixi/experiment/pytorchexperiment.py", line 19, in <module>
    from trixi.logger.message.slackmessagelogger import SlackMessageLogger
  File "/home/florian/.local/lib/python3.6/site-packages/trixi/logger/message/slackmessagelogger.py", line 4, in <module>
    from slackclient import SlackClient
ModuleNotFoundError: No module named 'slackclient'
```

**Comment by elpequeno** [comment]
Hey meowlz,

it seems that there is a problem in the requirements of trixi. There `slackclient==1.3.1` is used, which does not work for us. I recommend to specifically upgrade using:

`pip install slackclient==2.0.0`

I reported this issue to the trixi developers. I'll fix it here by adding the above requirement to the basic_unet_example requirements.txt

Let me know if it works.

**Comment by neuronflow** [comment]
Thank you, it seems that your change fixed it and it seems to be training now :)
However nothing shows up in the visdom dashboard if I refresh it?
I guess the network training should show up there?

**Comment by neuronflow** [comment]
Training finished I now see results in training and on the 2nd time training could see it live in visdom...after training it always says:
`Testing ended. Results stored =)
INFO:default-mWmI1zL5p0:Testing ended. Results stored =)
Testing complete.
ERROR:visdom:Connection is already closed.`

**Comment by elpequeno** [comment]
The empty dashboard is caused by the fact, that the visdom server needs to be started once to be able to download some needed scripts. This needs to be done only once (it is mentioned in the readme). THe second time it should work normally.

You can ignore the `ERROR:visdom:Connection is already closed` as long as your training finishes without errors and your results are stored properly.

Looking forward to your feedback on our repo.
If the is nothing else, I would close this issue.

**Comment by elpequeno** [comment]
I'll close this one. Please open up a new issue if you encounter more difficulties.
